### ISSUE TITLE
Again my app stuck in "on going" process ? Do I need to restart everyday?

### ISSUE THREAD

**Issue by himanshuteotia** [wit-ai/wit#1257]
![image](https://user-images.githubusercontent.com/10083136/47902437-0478a200-debd-11e8-982a-81a9e085dc20.png)

App ID : 2329833933697368

**Comment by blandinw** [comment]
@himanshuteotia could you elaborate what you mean by "restart"? Did you experience this exact issue before?

For now, it looks like your app has been trained successfully.

**Comment by asfyra** [comment]
I have the same issue [again](https://github.com/wit-ai/wit/issues/1248) and [again](https://github.com/wit-ai/wit/issues/1248#issuecomment-434194475)
AppId: 5b1e505b-0e5f-484a-9e44-20d7c4ee12d1


Today it's Novenber 4
![image](https://user-images.githubusercontent.com/6898230/47962214-01e38d00-e022-11e8-93d7-1149068a1f4b.png)


And the last train was on 2 of November 
![image](https://user-images.githubusercontent.com/6898230/47962216-16278a00-e022-11e8-9389-c412645eb9ea.png)

It looks like the app is stuck for 2 days now
Is there a way to restart the project ?

Thank you

**Comment by himanshuteotia** [comment]
Yeah, I also have the same issue ... even I just have right now 10 intents, for that I have to come here daily to restart my app.

**Comment by asfyra** [comment]
@blandinw you are right, we have a large dataset, but as the screenshot reveals it takes about 30 minutes to finish. Right now the app is stuck. The training cannot finish. The last sucessful one was on Friday.
@himanshuteotia how to you restart the app?

Thank you

**Comment by himanshuteotia** [comment]
@asfyra ... I have to create issue here then someone from here just restart my server. One more thing I want to menstion here  I trained my app today at around 12 PM and now it scheduled for 7 PM tonight.  I don't understand why it needs too much time for training. 
![screenshot from 2018-11-05 15-14-19](https://user-images.githubusercontent.com/10083136/47983393-1bfa9980-e10e-11e8-8a49-2b95ff1d8359.png)

**Comment by asfyra** [comment]
Can somebody please restart our application (AppId: 5b1e505b-0e5f-484a-9e44-20d7c4ee12d1)
It's stuck since last Friday (11/2/2018)

Thanks in advance for your effort
Alex

**Comment by asfyra** [comment]
After my previous post, someone must have restated  the application because it started training normally again with no problems but not for long 

Since this morning the application stack again !

{
   "name": "WinFAQV2_gr",
   "id": "5b1e505b-0e5f-484a-9e44-20d7c4ee12d1",
   "description": "FAQ bot",
   "lang": "el",
   "private": true,
   "created_at": "2018-06-11T03:35:07-0700",
   "last_training_duration_secs": 540,
   "will_train_at": "2018-11-06T00:01:49-0800",
   "last_trained_at": "2018-11-05T23:29:55-0800",
   "training_status": "ongoing"
}

Could you please reset the app for one more time. Is there something that can be done from my side to prevent this issue?

Thank you 
Alex

**Comment by blandinw** [comment]
@asfyra thank you for your feedback!
The training workers for your app were intermittently running out of memory. I'll be pushing a fix later today.

@himanshuteotia you may be experiencing the same issue. Nobody restarted these apps, but the system can unblock itself now, which is probably what has happened.

I'll close this issue once the fix is deployed later today. Please open a new issue if this problem happens again in the future.

**Comment by asfyra** [comment]
@blandinw Thank you very much indeed for your help.
